tutorials/discussions/plot_background_filtering.py

```diff
@@ -619,28 +619,28 @@ def plot_signal(x, offset):
 # ------------------
 #
 # When it comes to high-pass filtering, using corner frequencies above 0.1 Hz
-# were found in Acunzo *et al.* 2012 [4]_ to:
+# were found in Acunzo *et al.* (2012) [4]_ to:
 #
-#    "...generate a systematic bias easily leading to misinterpretations of
+#    "... generate a systematic bias easily leading to misinterpretations of
 #    neural activity.”
 #
-# In a related paper, Widmann *et al.* 2015 [7]_ also came to suggest a 0.1 Hz
-# highpass. And more evidence followed in Tanner *et al.* 2015 [8]_ of such
-# distortions. Using data from language ERP studies of semantic and syntactic
-# processing (i.e., N400 and P600), using a high-pass above 0.3 Hz caused
-# significant effects to be introduced implausibly early when compared to the
-# unfiltered data. From this, the authors suggested the optimal high-pass
-# value for language processing to be 0.1 Hz.
+# In a related paper, Widmann *et al.* (2015) [7]_ also came to suggest a
+# 0.1 Hz highpass. More evidence followed in Tanner *et al.* (2015) [8]_ of
+# such distortions. Using data from language ERP studies of semantic and
+# syntactic processing (i.e., N400 and P600), using a high-pass above 0.3 Hz
+# caused significant effects to be introduced implausibly early when compared
+# to the unfiltered data. From this, the authors suggested the optimal
+# high-pass value for language processing to be 0.1 Hz.
 #
-# We can recreate a problematic simulation from Tanner *et al.* 2015 [8]_:
+# We can recreate a problematic simulation from Tanner *et al.* (2015) [8]_:
 #
 #    "The simulated component is a single-cycle cosine wave with an amplitude
-#    of 5µV, onset of 500 ms poststimulus, and duration of 800 ms. The
+#    of 5 µV, onset of 500 ms poststimulus, and duration of 800 ms. The
```

Comment:
-1 on modifying direct quotes (the thing to do here if you want to say "we know it's on error but it's in the original" is to put `[sic]`)